jplag/src/main/java/de/jplag/strategy/AbstractComparisonStrategy.java

```diff
@@ -11,10 +10,6 @@
 
 public abstract class AbstractComparisonStrategy implements ComparisonStrategy {
 
-    // TODO PB: I think it's better to make each submission store its own matches with the base code.
-    // Hashtable that maps the name of a submissions to its matches with the provided base code.
-    private Hashtable<String, JPlagComparison> baseCodeMatches = new Hashtable<>(30);
```

Comment:
Ah, I see, it is falsely named here, so not your fault. Let us use this opportunity to fix it though.